src/test/java/unit/AwsSageMakerNotebookControlledTest.java

```diff
@@ -90,13 +91,17 @@ void listDescribeResolveReflectCreateDelete() throws IOException, InterruptedExc
     assertNotNull(createdResource.instanceType, "created resource returned instance type");
     assertNotNull(createdResource.instanceStatus, "created resource returned instance status");
 
-    // sagemaker notebooks are always private
+    // sagemaker notebooks are always private, no clone support
     assertEquals(
         AccessScope.PRIVATE_ACCESS, createdResource.accessScope, "created resource matches access");
     assertEquals(
         workspaceCreator.email.toLowerCase(),
         createdResource.privateUserName.toLowerCase(),
         "created resource matches private user name");
+    assertEquals(
```

Comment:
Would you mind adding this assertion into the test in GcpNotebookControlled.java? I had removed the existing assert rather than what you have here, which I think is correct.

Reply:
Sure, done